ioc/list.py

```diff
@@ -206,7 +231,10 @@ def _print_json(
 
 
 def _lookup_resource_values(
-    resource: 'iocage.Resource.Resource',
+    resource: typing.Union[
+        'iocage.Resource.Resource',
+        typing.Dict[str, str]
+    ],
     columns: typing.List[str]
 ) -> typing.List[str]:
     if "getstring" in resource.__dir__():
```

Comment:
flake8 error:
```
./ioc/list.py:242:33: T484 Item "Dict[str, str]" of "Union[Any, Dict[str, str]]" has no attribute "getstring"
```

Reply:
Flake8 is a merge requirement. No need to report those manually in reviews :)